modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeactivateKeyBeforeSelectingPortal.testcase

```diff
@@ -0,0 +1,117 @@
+@component-name = "portal-solutions"
+definition {
+
+	property osgi.modules.includes = "osb-site-initializer-customer-portal";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "OSB Site Initializer Customer Portal";
+
+	var testSiteName = "Customer Portal";
+	var testSiteURL = "customer-portal-site";
+	var testAccount = "2022 FIFA World Cup Qatar - FAN ID";
+	var statusActive = "Active";
+
+	setUp {
+		task ("Set up instance and sign in") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+			task ("Create a new Customer Portal Site") {
+				CustomerPortal.addCustomerPortalSite(siteName = ${testSiteName});
+			}
+
+			task ("Add test user access to all Accounts") {
+				CustomerPortal.assignUserToOrganization();
+
+				CustomerPortal.addUserToOneOrMoreAccounts(accountsList = ${testAccount});
+			}
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		CustomerPortal.customerPortalTearDown();
+
+		JSONGroup.deleteGroupByName(groupName = ${testSiteName});
+	}
+
+	@description = "LPS-185975 - Verify that after clicking the 'Deactivate' button, a page appears with a list of active keys"
+	@priority = 4
+	test AssertThatAfterClickInDeactivateAPageOpens {
+		task ("Given a user signs in on OKTA to update the Customer Portal") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal");
+		}
+
+		task ("When the user selects a project in Customer Portal and chooses to Deactivate") {
+			CustomerPortal.enterOnProject(
+				status = ${statusActive},
+				title = ${testAccount});
+
+			CP2NavigationMenu.goToProductActivation();
+
+			CP2NavigationMenu.clickOnProductButton(product = "Portal");
+		}
```

Comment:
In this task it says that he selects deactivate, looking through the macros, in this task only goes to the Portal product, is that right?